AUTHORS

```diff
@@ -735,5 +735,14 @@ Alex Yursha <alexyursha@gmail.com>
 Steven Vercruysse <stcruy@users.noreply.github.com>
 Aleksanteri Negru-Vode <aleksanteri.negru-vode@aalto.fi>
 Mathieu Darse <mattdarse@gmail.com>
+Connor Peet <connor@peet.io>
+Mayhem <stepien.nicolas@gmail.com>
+Olov Lassus <olov.lassus@gmail.com>
+Phillip Lamplugh <phillip.lamplugh@gmail.com>
+Kohei TAKATA <kt.koheitakata@gmail.com>
+Giovanny Andres Gongora Granada <gioyik@gmail.com>
+Jeffrey Jagoda <jeffrey.jagoda@gmail.com>
+Kelsey <ifoundthemeaningoflife@gmail.com>
```

Comment:
fixed in .mailmap so it's now showing up with your full name @Frijol, but you should still: `git config --add user.name 'Kelsey Breseman'